src/Symfony/Component/Console/Formatter/OutputFormatterInterface.php

```diff
@@ -62,8 +62,9 @@ public function getStyle($name);
      * Formats a message according to the given styles.
      *
      * @param string $message The message to style
+     * @param int    $width   The message width
      *
      * @return string The styled message
      */
-    public function format($message);
+    public function format($message/*, $width = 0*/);
```

Comment:
Any suggestions? Took this approach from  `LogoutUrlGenerator::registerListener` ;) which is not bound to an interface.

Note that this way is the easiest implementation and avoids double processing the formatted text, which is nice.